saleor/graphql/discount/tests/mutations/utils.py

```diff
@@ -0,0 +1,28 @@
+from collections import defaultdict
+from typing import DefaultDict, Set, Union
+
+CatalogueInfo = DefaultDict[str, Set[Union[int, str]]]
+PREDICATE_TO_CATALOGUE_INFO_MAP = {
+    "collectionPredicate": "collections",
+    "categoryPredicate": "categories",
+    "productPredicate": "products",
+    "variantPredicate": "variants",
+}
+
+
+def convert_migrated_sale_predicate_to_catalogue_info(
+    catalogue_predicate,
+) -> CatalogueInfo:
+    catalogue_info: CatalogueInfo = defaultdict(set)
+    for field in PREDICATE_TO_CATALOGUE_INFO_MAP.values():
+        catalogue_info[field] = set()
+
+    if catalogue_predicate.get("OR"):
+        predicates = {
+            list(item.keys())[0]: list(item.values())[0]["ids"]
```

Comment:
Please consider using `next` to get first value
```suggestion
            next(item.keys()): next(item.values())["ids"]
```